index.html

```diff
@@ -10,31 +11,202 @@
+        <li><button class="custom-button" type="button">html</button></li>
+        <li><button class="custom-button" type="button">css</button></li>
+        <li><button class="custom-button" type="button">JavaScript</button></li>
+      </ul>
+      <button class="big-button" type="button">See Project</button>
+    </div>
+
+    <div class="pageone">
+      <img class="card-image" src="./img/Snapshoot-portfolio2.svg" alt="portfolio-snapshoot1">
+      <h1 class="left">Multi-Post <br>
+        Stories</h1>
+      <div class="flex">
+        <h2>CANOPY</h2>
+        <ul class="flex-items">
+          <li>Black End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p class="left">A daily selection of privately <br>
+        personalized reads; no accounts or <br>
+        sign-ups required.</p>
+      <ul class="flex-button">
+        <li><button class="custom-button" type="button">html</button></li>
+        <li><button class="custom-button" type="button">css</button></li>
+        <li><button class="custom-button" type="button">JavaScript</button></li>
+      </ul>
+      <button class="big-button" type="button">See Projects</button>
+    </div>
+
+
+
+
+    <div class="pageone">
+      <img class="card-image" src="./img/Snapshoot-portfolio3.svg" alt="portfolio-snapshoot1">
+
+      <h1 class="left">Tonic</h1>
+      <div class="flex">
+        <h2>CANOPY</h2>
+        <ul class="flex-items">
+          <li>Black End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p class="left">A daily selection of privately <br>
+        personalized reads; no accounts or <br>
+        sign-ups required.</p>
+      <ul class="flex-button">
+        <li><button class="custom-button" type="button">html</button></li>
+        <li><button class="custom-button" type="button">css</button></li>
+        <li><button class="custom-button" type="button">nJavaScript</button></li>
+      </ul>
+      <button class="big-button" type="button">See Project</button>
+    </div>
+
+
+
+    <div class="pageone">
+      <img class="card-image" src="./img/Snapshoot-portfolio4.svg" alt="portfolio-snapshoot1">
+      <h1 class="left">Multi-Post <br>
+        Stories</h1>
+      <div class="flex">
+        <h2>CANOPY</h2>
+        <ul class="flex-items">
+          <li>Black End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p class="left">A daily selection of privately <br>
+        personalized reads; no accounts or <br>
+        sign-ups required.</p>
+      <ul class="flex-button">
+        <li><button class="custom-button" type="button">html</button></li>
+        <li><button class="custom-button" type="button">css</button></li>
+        <li><button class="custom-button" type="button">JavaScript</button></li>
+      </ul>
+      <button class="big-button" type="button">See Project</button>
     </div>
-    <h2 class="blue-text">LET'S CONNECT</h2>
+  </section>
+  <section class="about-me">
```

Comment:
- [ ] [OPTIONAL] Kindly let's give a bit of `padding-top` to the  about-me section because for now, the content is stack to the top 👍

Reply:
Thank you so much, I will fix it.